### PR TITLE
housekeeping

### PR DESCRIPTION
**Pull request by eseiker**
- merged deno.json in web directory into parent directory's deno.jsonc
- use import map for packages used more than once (except viem imports in decodeEventLog.ts)
- tried to annotate type only imports, organize imports as much as possible
- deno fmt

any suggestions welcome

### CODE REVIEW

File: api.ts [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
+
+import { oakCors } from "https://deno.land/x/cors@v1.2.2/mod.ts";
 
 import { Buffer } from "node:buffer";
 
 import { stringify as losslessJsonStringify } from "npm:lossless-json";
 import { getAddress, keccak256, toHex } from "npm:viem";
 
 import type { PrismaClient } from "./prisma-shim.ts";
-
```
Comment (tkiapril) [inline api.ts:22]: The reason why I had prisma imports separate from others was that it has a bit different purpose than the other local imports, in which it's 1) generated, 2) a shim, and 3) closer to a library. What's your thought on these premises?

File: decodeEventLog.ts [hunk 2]
```diff
@@ -4,7 +4,6 @@ import type {
   ExtractAbiEventNames,
   Narrow,
 } from "https://esm.sh/abitype";
-
```
Comment (tkiapril) [inline decodeEventLog.ts:7]: This empty line was to separate typical imports and partial hacky imports, what do you think about this?

File: deno.jsonc [hunk 3]
```diff
@@ -3,7 +3,31 @@
...
   },
-  "lock": false
+  "lock": false,
+  "compilerOptions": {
+    "jsxImportSource": "preact",
+    "jsx": "react-jsx"
+  },
+  "imports": {
+    "~root/": "./",
+    "~/": "./web/",
```
Comment (tkiapril) [inline deno.jsonc:16]: Might it make more sense to have "~/" as "./" and "web/" as "./web/"?

File: emitter.ts [hunk 4]
```diff
@@ -18,7 +15,6 @@ import {
 } from "npm:viem";
 
 import type { PrismaClient } from "./prisma-shim.ts";
-
```
Comment (tkiapril) [inline emitter.ts:21]: Same here about prisma imports.

**Review by tkiapril — COMMENTED** [review]
What's your stance on having imports with "npm:" qualifiers in import maps as well? I personally don't have a preference, but if we're to make a convention, it might matter to stick to a practice.

**Comment by eseiker** [comment]
In my preference, npm packages are not subjectives to be listed in import map.

**Review by tkiapril — APPROVED** [review]

**Pull request merged**